kubetel.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
let's not push this config to open source repo.. as it contains sqs endpoint.. which we do not want to exposure to external.

Reply:
Will remove